tests/e2e/e2e_suite_test.go

```diff
@@ -35,7 +32,6 @@ func TestE2e(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	ctx, cancel = context.WithCancel(context.TODO())
-	logf.SetLogger(log.ConfigLogger(9, zapcore.AddSync(GinkgoWriter)))
```

Comment:
why remove this?

Reply:
We are not logging anything using this one. Also this is the controller-runtime logger

Reply:
Yes, but the `envtest` is from controller-runtime, it's fine we adjust the logger setting here. But I got it, the `ConfigLogger` is coming from lifecycle-manager, it's crazy we cross-reference like this.